sql/resource_sql_endpoint.go

```diff
@@ -99,9 +99,13 @@ type SQLEndpointsAPI struct {
 }
 
 // List all SQL endpoints
-func (a SQLEndpointsAPI) List() (l EndpointList, err error) {
-	err = a.client.Get(a.context, "/sql/endpoints", nil, &l)
-	return
+func (a SQLEndpointsAPI) List() ([]SQLEndpoint, error) {
+	var lst endpointList
```

Comment:
Newer API wrappers have listing methods have struct returned, not arrays. Let's keep consistency.